src/hist_smooth/grid_plot_hist.py

```diff
@@ -1,10 +1,12 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import hist
-from smoothing_lib.smoothing_api import Smoothing
-from smoothing_lib.statistics_utils import reduced_chi2, ks_2samp
+from hist_smooth.smoothing_api import smooth_hist_general
+from hist_smooth.statistics_utils import reduced_chi2, ks_2samp
+import boost_histogram as bh
 
-def plot_grid_with_smoothing(data_list, binnings, algorithms, figsize=(5, 4)):
+def plot_grid_with_smoothing(data_list, binnings, algorithms, figsize=(5, 4), hsys_hist=None,

```

Comment:
will this function be exposed in the API to the user? if so should it not just accept a `hist` object?